src/runtime_src/core/common/xclbin_parser.cpp

```diff
@@ -387,5 +391,57 @@ get_kernel_freq(const axlf* top)
   return kernel_clk_freq;
 }
 
+
+std::vector<uint64_t>
+get_kernel_inst_addrs(const axlf* top)
+{
+  std::vector<uint64_t> addrVec;
```

Comment:
Can you please add a comment for this code saying that ,this code is going to be removed once rtd file is generated for sw_emu.

Reply:
We are planning to remove the EMBEDDED_METADATA section usage in 2020.1+

Reply:
comment added in xclbin_parser.cpp